inc/namespace.php

```diff
@@ -818,7 +818,9 @@ function get_user_roles_from_sso( \WP_User $user, array $attributes ) {
 		} elseif ( ! isset( $roles['network'] ) && ! isset( $roles['sites'] ) ) { // Associative but no 'network' or 'sites' keys ?
 			$network_roles = [];
 		}
+
+		return $network_roles;
 	}
 
-	return $network_roles ?? (array) $roles;
+	return $roles;
```

Comment:
Why is this change here? There shouldn't be any functional difference?

Reply:
It was just weirdly written, `$network_roles` is only defined in the `if` block above this return statement, and `$roles` is already cast to an array.